doc/controls/ZoomContentControl.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: Toolkit.Controls.ZoomContentControl
+---
+
+# ZoomContentControl
+
+> [!TIP]
+> This guide covers details for the `ZoomContentControl`. If you are just getting started with the Uno Toolkit UI Library, please see our [general getting started](../getting-started.md) page to make sure you have the correct setup in place.
+
+## Summary
+
+`ZoomContentControl` allows you to display content that can be zoomed in and out, as well as panned. It is especially useful for scenarios such as viewing large images, maps, or documents where users need control over zoom levels and panning.
+
+### C\#
+
+```csharp
+public partial class ZoomContentControl : Control
+```
+
+### XAML
+
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:ZoomContentControl ZoomLevel="1.5"
+                        MinZoomLevel="0.5"
+                        MaxZoomLevel="3.0"
+                        IsZoomAllowed="True"
+                        IsPanAllowed="True">
+    <utu:ZoomContentControl.Content>
+        <Image Source="ms-appx:///Assets/Media/LargeMedia.svg"
+               Stretch="Uniform" />
+    </utu:ZoomContentControl.Content>
+</utu:ZoomContentControl>
+```
+
+### Inheritance
+
+`Object` &#8594; `DependencyObject` &#8594; `UIElement` &#8594; `FrameworkElement` &#8594; `Control` &#8594; `ContentControl` &#8594; `ZoomContentControl`
+
+### Constructors
+
+| Constructor| Description|
+|----------------|-------------------------------------------------------|
+| `ZoomContentControl()`|Initializes a new instance of the `ZoomContentControl` class.|
+
+### Properties
+
+| Property| Type| Description|
+|------------------------|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `ZoomLevel`| `double`| Gets or sets the current zoom level for the content.|
```

Comment:
we discussed about the formatting, you should either format to align the table, or
just use single - for header separator and left single space on each side of cell content:
```suggestion
| Property| Type | Description |
|-|-|-|
| `ZoomLevel`| `double` | Gets or sets the current zoom level for the content. |
```